### PR TITLE
Restore core unit tests

### PR DESCRIPTION
**Pull request by ghost**
Restored "core" tests.
Some noise in `dll.c' need be fixed,
but I think it would be nice to have this up and running.

### CODE REVIEW

File: unittest.sh [hunk 1]
```diff
@@ -14,3 +14,4 @@ if [ -n "$DEBUG" ] ; then
 fi
 
 /usr/bin/php support/unittester/run-saffire-tests.php ${1:-"unittests/tests/"}
+./build/release/utmain
```
Comment (jaytaph) [inline unittest.sh:16]: Would it make more sense to start with the core tests first? (ie: swap these two lines?)

Comment (ghost) [inline unittest.sh:16]: Yes, sure!

**Comment by jaytaph** [comment]
thanks @steazzalini

**Comment by ghost** [comment]
de nada @jaytaph  :beers:

**Pull request merged**